src/Components/Artwork/SaveButton/SaveButton.tsx

```diff
@@ -43,31 +54,39 @@ export const SaveButtonBase: React.FC<SaveButtonBaseProps> = ({
   }
 
   return (
-    <Clickable
-      aria-label={isSaved ? "Unsave" : "Save"}
-      data-test="saveButton"
-      height={BTN_HEIGHT}
-      width={BTN_WIDTH}
-      onMouseEnter={handleMouseEnter}
-      onMouseLeave={handleMouseLeave}
-      onClick={onClick}
-    >
-      {isSaved || isHovered ? (
-        <HeartFillIcon
-          title={title}
-          fill="blue100"
-          width={BTN_WIDTH}
-          height={BTN_HEIGHT}
-        />
-      ) : (
-        <HeartStrokeIcon
-          title={title}
-          fill="black100"
-          width={BTN_WIDTH}
-          height={BTN_HEIGHT}
-        />
+    <Flex alignItems="center">
+      {shouldDisplayLotCount && (
+        <Text variant="xs" color="black100">
+          {lotWatcherCount}
+        </Text>
       )}
-    </Clickable>
+
+      <Clickable
+        aria-label={isSaved ? "Unsave" : "Save"}
+        data-test="saveButton"
+        height={BTN_HEIGHT}
+        width={BTN_WIDTH}
+        onMouseEnter={handleMouseEnter}
+        onMouseLeave={handleMouseLeave}
+        onClick={onClick}
+      >
+        {isSaved || isHovered ? (
+          <HeartFillIcon
+            title={title}
+            fill="blue100"
+            width={BTN_WIDTH}
+            height={BTN_HEIGHT}
+          />
+        ) : (
+          <HeartStrokeIcon
+            title={title}
+            fill="black100"
+            width={BTN_WIDTH}
+            height={BTN_HEIGHT}
+          />
+        )}
```

Comment:
non-blocking: do these need to be repeated or will clickable automatically wrap whichever icon is rendered?